.github/workflows/node-build.yml

```diff
@@ -134,6 +134,29 @@ jobs:
      - name: Perform CodeQL Analysis
        uses: github/codeql-action/analyze@v3
 
+  integration-test:
```

Comment:
I think we should add this to the `needs` list for the `node-build` -> we should probably not run node-build if we fail integration tests.

CC @BlairCurrey

Reply:
yeah I'd say so too.

Reply:
Done